content/run.md

```diff
@@ -240,3 +240,111 @@ Now you can perform the final steps to build and submit the case to the job queu
 
     [~/cases/SCA_SpRun]$ ./case.build
     [~/cases/SCA_SpRun]$ ./case.submit
+
+### Example regional run bash scripts
```

Comment:
Would it be an idea to instead have a general header at the bottom of "[Run CTSM](https://metos-uio.github.io/CTSM-Norway-Documentation/run/#run-ctsm)" called "Example scripts"? Then we can add several tabs with scripts for f.ex. single site etc.

Reply:
Yes, the "Example scripts" is now at the same level as "Run a regional case" (f84345dc1419812772d3512e36609d6b8faf4109)